adala/runtimes/_litellm.py

```diff
@@ -539,7 +539,14 @@ async def batch_to_batch(
             )
 
         extra_fields = extra_fields or {}
+        input_field_types = input_field_types or {}
         records = batch.to_dict(orient="records")
+        # in multi-image cases, the number of tokens can be too large for the context window
+        # so we need to split the payloads into chunks
```

Comment:
add comment - this may happen for any project type, just more likely in this case, so this setting is a heuristic

Reply:
done